packages/klevu-core/src/connection/klevuFetch.ts

```diff
@@ -1,6 +1,6 @@
 import { FetchResultEvents } from "../events/FetchResultEvents.js"
 import {
-  applyFilterWithManager,
+  applyFilterWithManager, KlevuAnnotations,
```

Comment:
Looks like prettier is not run on this file. It's important to keep things to look the same.